src/views/SubmissionView/TextResponseEditor/__snapshots__/RichTextEditor.test.jsx.snap

```diff
@@ -17,13 +17,14 @@ exports[`<RichTextEditor /> render disabled 1`] = `
     disabled={true}
     init={
       Object {
-        "content_css": false,
+        "base_url": "http://localhost:18000/static/js/vendor/tinymce/js/tinymce",
```

Comment:
implies the localhost LMS is running in the environment.  I don't believe this is going to be true in all situations.

Reply:
This is a snapshot. That url came from test data.